@xen-orchestra/lite/src/App/TabConsole.tsx

```diff
@@ -52,28 +52,44 @@ const TabConsole = withState<State, Props, Effects, Computed, ParentState, Paren
       setCtrlAltDel: function (sendCtrlAltDel) {
         this.state.sendCtrlAltDel = sendCtrlAltDel
       },
+      showNotImplemented: function () {
+        alert('Not Implemented')
+      },
     },
     computed: {
       vm: (state, { vmId }) => state.objectsByType.get('VM')?.get(vmId),
     },
   },
   ({ effects, state, vmId }) => (
     <div style={{ height: '100%' }}>
-      {state.vm?.power_state !== 'Running' ? (
-        <p>
-          <IntlMessage id='consoleNotAvailable' />
-        </p>
-      ) : (
-        <>
-          {/* Hide scaling and Ctrl+Alt+Del button temporarily */}
-          {/* <RangeInput max={100} min={1} onChange={effects.scaleConsole} step={1} value={state.consoleScale} />
+      <PanelHeader
+        actions={[
+          {
+            key: 'start',
+            icon: 'play',
+            color: 'primary',
+            title: translate({ id: 'vmStartLabel' }),
+            variant: 'contained',
+            onClick: effects.showNotImplemented,
+          },
+        ]}
+      >
+        {state.vm?.name_label ?? 'loading'}{' '}
+      </PanelHeader>
+
+      {/* Hide scaling and Ctrl+Alt+Del button temporarily */}
+      {/* <RangeInput max={100} min={1} onChange={effects.scaleConsole} step={1} value={state.consoleScale} />
           {state.sendCtrlAltDel !== undefined && (
             <Button onClick={state.sendCtrlAltDel}>
               <IntlMessage id='ctrlAltDel' />
             </Button>
           )} */}
-          <Console vmId={vmId} scale={state.consoleScale} setCtrlAltDel={effects.setCtrlAltDel} />
-        </>
+      {state.vm?.power_state !== 'Running' ? (
+        <p>
+          <IntlMessage id='consoleNotAvailable' />
+        </p>
+      ) : (
+        <Console vmId={vmId} scale={state.consoleScale} setCtrlAltDel={effects.setCtrlAltDel} />
```

Comment:
It looks like the bottom of the console is cropped when I test it. Do you reproduce?

Reply:
yes. I think it comes from the way the console size is computed. Can I fix it in this PR ( with a litlle of flex magic) ? ou should I make a separated PR